packages/core/src/baseclient.ts

```diff
@@ -713,6 +713,16 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
    * @param scope
    */
   protected _captureEvent(event: Event, hint: EventHint = {}, scope?: Scope): PromiseLike<string | undefined> {
+    if (DEBUG_BUILD && isErrorEvent(event)) {
+      logger.log(
+        `Captured error event \`${
```

Comment:
This is a bit annoying/verbose, but not sure if/how we can streamline this - do we have an existing method somewhere that does this already? 🤔

Reply:
We have `_getPossibleEventMessages` in the inbound filters integration and/or somewhat related `_getExceptionFromEvent` in the dedupe integration.

Reply:
Ah nice, I'll use `getPossibleEventMessages`!